monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy

```diff
@@ -24,9 +26,18 @@ public class MonkeyConfigurationPlugin implements Plugin<Project> {
     }
 
     private static void ensureAndroidPluginAppliedTo(Project project) {
-        boolean missingAndroidPlugin = !project.plugins.hasPlugin('com.android.application')
-        if (missingAndroidPlugin) {
-            throw new GradleException('monkey runner plugin can only be applied after the Android plugin')
+        ensurePluginIsApplied('com.android.application', project)
+    }
+
+    private static void ensureCommandPluginAppliedTo(Project project) {
+        ensurePluginIsApplied('android-command', project)
```

Comment:
Our plugin can also be applied like `com.novoda.android-command`. 
I believe this will still work even the user applied the FQN (because gradle magic?)